src/package-operations.ts

```diff
@@ -167,15 +179,19 @@ export async function updatePackage(
  */
 function getUpdatedManifest(
   currentManifest: Partial<PackageManifest>,
-  updateSpecification: UpdateSpecification,
+  updateSpecification: UpdateSpecification &
+    Partial<MonorepoUpdateSpecification>,
```

Comment:
Why `& Partial<>`? It seems that this would be more correct

```suggestion
  updateSpecification: UpdateSpecification |
    MonorepoUpdateSpecification,
```